waarg-overrides.nix

```diff
@@ -0,0 +1,22 @@
+sources: hlib: hself: hsuper: {
+  # doctest = hsuper.callCabal2nix "doctest" sources.doctest {};
+  # haskellworks
+  hw-bits = hsuper.callCabal2nix "hw-bits" sources.hw-bits {};
+  hw-balancedparens = hlib.dontCheck (
+    hsuper.callCabal2nix "hw-balancedparens" sources.hw-balancedparens {}
+  );
+  hw-json-simd = hlib.markUnbroken hsuper.hw-json-simd;
```

Comment:
amended: `hw-json-simd` is marekd as broken downstream. I've marked it unbroken here.